src/app/pages/services/components/service-ups/service-ups.component.ts

```diff
@@ -268,11 +268,12 @@ export class ServiceUPSComponent implements OnDestroy {
         _.find(this.fieldConfig, { name: 'ups_driver' }).isHidden = true;
         _.find(this.fieldConfig, { name: 'ups_port' }).isHidden = true;
         _.find(this.fieldConfig, { name: 'ups_remotehost' }).isHidden = false;
-        _.find(this.fieldConfig, { name: 'ups_remoteport' }).isHidden = false;
         _.find(this.fieldConfig, { name: 'ups_options' }).isHidden = true;
       } else {
         this.entityForm.setDisabled('ups_driver', false);
         this.entityForm.setDisabled('ups_port', false);
+        this.entityForm.setDisabled('ups_remotehost', true);
+        this.entityForm.setDisabled('ups_remotehost', true);
```

Comment:
change one of these lines to read 
this.entityForm.setDisabled('ups_remoteport', true);